subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -660,7 +660,7 @@ config BT_MESH_SCENES_MAX
 	help
 	  Maximum number of scenes that can be stored by a single Scene Server.
 	  According to the specification, this value should be 16.
-	  The Bluetooth Mesh Model specification v1.0.1 (MshMDLv1.0.1) defines the
+	  The Bluetooth Mesh Model specification v1.1 (MshMDLv1.1) defines the
```

Comment:
Do you feel strongly about this? I already changed this format once before upon request, and it spans over both the PR for upstream and NCS. If you think this must be changed we need to discuss it among our devs internally so that there is a consensus.

Reply:
As this has been done in the same way every where I think we should keep it the way it is.

Reply:
It is already merged for upstream... If this is really that important I suggest to create a separate task/PR for it.

Reply:
Meh .. ok.